aesm-client/tests/live_quote.rs

```diff
@@ -41,3 +44,38 @@ fn live_quote() {
         )
         .expect("quote result");
 }
+
+#[test]
+fn test_ecdsa_quote() {
+    // Doing a meaningful test of this requires creating an enclave, this is
+    // just a simple test that we can send a bogus request and get an error
+    // back. The node attest flow in testsetup.sh exercises the real case.
```

Comment:
Uh, what now?

Reply:
I copied that comment - need to remove it, it's not relevant :)